dc/core/ContentWindow.cpp

```diff
@@ -203,48 +236,9 @@ bool ContentWindow::isHidden() const
     return windowState_ == HIDDEN;
 }
 
-bool ContentWindow::registerEventReceiver( deflect::EventReceiver* receiver )
+ContentInteractionDelegate* ContentWindow::getInteractionDelegate()
```

Comment:
I'm also wondering here, because there is no delegate if no Content has been assigned to the window